node/consensus/consensus.go

```diff
@@ -160,9 +160,20 @@ func (c *Consensus) SubmitConfig(ctx context.Context, request *protos.Request) (
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to verify and classify request")
 	}
-	_ = configRequest
-	// TODO: submit configRequest.Bytes()
-	return &protos.SubmitResponse{Error: "SubmitConfig is not implemented in consenter", TraceId: request.TraceId}, nil
+
+	if configRequest == nil {
+		return nil, errors.Errorf("unexpected config request was verified and proposed")
+	}
```

Comment:
In line 152 above you are verifying the request an generating a `configRequest`. I expect the method to never return a nil object unless `err!=nil`.
This check is redundant, or at least push it inside `verifyAndClassifyRequest`.